modules/ipi-install-network-requirements.adoc

```diff
@@ -26,25 +28,166 @@ When using a VLAN, each NIC must be on a separate VLAN corresponding to the appr
 ====
 
 [discrete]
-== Configuring the DNS server
+== DNS requirements
 
 Clients access the {product-title} cluster nodes over the `baremetal` network. A network administrator must configure a subdomain or subzone where the canonical name extension is the cluster name.
 
+[source,text]
 ----
 <cluster_name>.<domain>
 ----
 
 For example:
 
+[source,text]
 ----
 test-cluster.example.com
 ----
 
 {product-title} includes functionality that uses cluster membership information to generate A/AAAA records. This resolves the node names to their IP addresses. After the nodes are registered with the API, the cluster can disperse node information without using CoreDNS-mDNS. This eliminates the network traffic associated with multicast DNS.
 
+In {product-title} deployments, DNS name resolution is required for the following components:
+
+* The Kubernetes API
+* The {product-title} application wildcard ingress API
+
+Reverse DNS resolution is also required for the Kubernetes API, the provisioner node, the control plane nodes, and the worker nodes.
```

Comment:
Yeah AFAIK reverse dns is only *required* in cases where the hostname isn't set via DHCP (because NetworkManager by default will try both methods to detect and set the hostname, and the hostname has to be derived somehow for CSR auto-approval to work)